test/lib/testAttributeEditorTemplate.py

```diff
@@ -201,11 +201,16 @@ def testAECustomMaterialControl(self):
         self.assertTrue(cmds.formLayout(meshFormLayout, exists=True))
         startLayout = cmds.formLayout(meshFormLayout, query=True, fullPathName=True)
         self.assertIsNotNone(startLayout, 'Could not get full path for Mesh formLayout')
-        
+
+        # In the AE there is a formLayout for each USD prim type. We start
+        # by making sure we can find the one for the mesh.
+        materialFormLayout = self.searchForMayaControl(startLayout, cmds.frameLayout, 'Material')
```

Comment:
I suspect you want `getMayaUsdLibString('kLabelMaterial')` instead of `'Material'`